tests/modules/aclcheck.c

```diff
@@ -215,12 +219,25 @@ int commandBlockCheck(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
 }
 
 int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
-    REDISMODULE_NOT_USED(argv);
-    REDISMODULE_NOT_USED(argc);
 
     if (RedisModule_Init(ctx,"aclcheck",1,REDISMODULE_APIVER_1)== REDISMODULE_ERR)
         return REDISMODULE_ERR;
 
+    if (argc != 1) return RedisModule_WrongArity(ctx);
+
+    long long fail_flag = 0;
+    RedisModule_StringToLongLong(argv[0], &fail_flag);
+    if (fail_flag) {
```

Comment:
let's add a quick comment that documents this block.
i.e. that when that flag is passed we create too many categories, and the test expects this to fail

Reply:
p.s. it would be nice to let the module handle a case of argc==0, and then we won't need to change all the other module load commands in the test code